app/code/Magento/Store/Api/Data/StoreConfigInterface.php

```diff
@@ -7,7 +7,7 @@
 
 /**
  * StoreConfig interface
- *
+ * Interface for store config
```

Comment:
For each of these where you update the class description, can you remove the old, useless description. and add a new line
so for this one it would be
```
/**
 * Interface for store config
 *
 * @api
```